mcloud.yaml

```diff
@@ -10,8 +10,7 @@ integrations:
     git_branch: $UV_BRANCH
     pip_install: poetry==1.7.1
 command: >-
-  cd ultravox && poetry install --no-dev && poetry run torchrun --nproc_per_node=8 -m ultravox.training.train --config_path ultravox/training/configs/$UV_CONFIG
+  cd ultravox && poetry install --no-dev && poetry run torchrun --nproc_per_node=8 -m ultravox.training.train --config_path ultravox/training/configs/llama3_whisper.yaml
```

Comment:
Since we can't control `UV_CONFIG` from the env variables, I'm not sure if it has any use at the moment.
I also changed the default to whisper.

Reply:
I think it is similar to UV_BRANCH, it is cleaner and easier to edit when they are separated from the rest.

Reply:
I don't like `UV_BRANCH` either, it's just a level of indirection that doesn't do anything. Though that one is not as annoying.
Also we don't use `MLFLOW_TRACKING_URI` any more.

Reply:
For new users, this signifies that these are typical changes you can make to run custom experiments, and it is easy to understand. The same can be said about git_repo.

Reply:
Yeah I mentioned that above, but `UV_CONFIG` doesn't even do that well, since it's saying you have to use one of the configs, but we never use it like that.

If we could change it to: `UV_ARGS` and let it include all arguments passed to the Python code it would be good, but I'm not sure if that would work correctly out of the box (e.g. the whole env var could be interpreted as a single argument, not sure).

Reply:
I guess it is not a big deal. The mcloud.yaml is more about providing an example on how to train a model at this point. I will take it.

Reply:
Alright, I modified this: now there's a `TRAIN_ARGS` that you can fill up instead of `UV_CONFIG`. It contains all of the args.